EssayMonkey.py

```diff
@@ -0,0 +1,87 @@
+# Thomas Doan
+# EssayMonkey.py
+# 10/30/15
+# Creates essay based on user given paragraph and sentence amount.
+
+import random
+
+'''
+Main function accepts user amount of paragraphs and sentences.
+Calls auxiliary function to create essay based on these amounts.
+'''
+
+"""
+Reads from the provided text files and makes lists with the words
+in the files. Using these lists, makes an "essay" with user given
+paragraph and sentence amount.
+Args:
+numParagraphs: The user given amount of paragraphs desired in essay.
+numSentences: The user given amount of sentences desired in essay.
+Returns:
+A randomly generated essay string with the given amount
+of paragraphs and sentences.
+"""
+
+
+def essayMonkey(numParagraphs,numSentences):
```

Comment:
Missing a a space after the comma. No big deal. Just pointing it out for the sake of consistency to your other functions.